src/main/java/com/iota/iri/model/safe/ByteSafe.java

```diff
@@ -1,5 +1,8 @@
 package com.iota.iri.model.safe;
 
+/**
+ *Utilises the HashSafeObject class to generate a null safe byte array with retrievable hash code index
```

Comment:
mention double-check locking.
It is more correct to write "extends the HashSafeObject", however this information will be automatically generated by the doc so no need to write anything about that.